### PR TITLE
Server setup via command-line arguments or environment arguments

### PR DESCRIPTION
**Pull request by PavloKovalov**
:squid:

### CODE REVIEW

**Comment by ai** [comment]
Looks good. I will look closer in next few hours.

File: README.md [hunk 1]
```diff
@@ -71,6 +71,17 @@ if (app.env === 'production') {
 }
 ```
 
+### Use console arguments or environment variables to redefine options
```
Comment (ai) [inline README.md]: No environment variables in docs and `--help` :)

File: base-server.js [hunk 2]
```diff
@@ -8,6 +8,29 @@ var http = require('http')
 var path = require('path')
 var Log = require('logux-core').Log
 var fs = require('fs')
+var yargs = require('yargs')
+  .option('h', {
+    alias: 'host',
+    describe: 'IP-address to bind server',
```
Comment (ai) [inline base-server.js]: Host is not equal to IP-address. `0.0.0.0` is not valid IP-address :).

File: base-server.js [hunk 3]
```diff
@@ -8,6 +8,29 @@ var http = require('http')
...
+    type: 'number'
+  })
+  .option('k', {
+    alias: 'key',
+    describe: 'SSL key or path to it',
+    type: 'string'
+  })
+  .option('c', {
+    alias: 'cert',
+    describe: 'SSL certificate or path to it',
```
Comment (ai) [inline base-server.js]: In CLI you can’t set file content directly. You will set path anyway.

File: test/base-server.test.js [hunk 4]
```diff
@@ -41,9 +41,19 @@ function createReporter (test) {
   })
 }
 
+function envCleanup () {
```
Comment (ai) [inline test/base-server.test.js]: You don’t need function to use it only once :)

File: README.md [hunk 5]
```diff
@@ -71,6 +71,17 @@ if (app.env === 'production') {
...
+```sh
+$ node base-server.js --help
```
Comment (ai) [inline README.md]: There is no `base-server.js`. You call it with `server.js --help`.

File: base-server.js (hunk 2 above)
Comment (PavloKovalov) [inline base-server.js]: Took description from JSDoc of BaseServer#listen

**Comment by ai** [comment]
Great. Thanks for fixes.

**Pull request merged**